### ISSUE TITLE
I am not allowed to put in an exact number of acres in Title

### ISSUE THREAD

**Issue by KelleyDrake** [ddrake/Integrated_Farm_Budget_Tool#186]
I am trying to get my model back to the right point in the tutorial sequence to then redo my SCO ECO tutorial.    I originally set up title on my Madison County farm as Corn 850 acres in PLC, Beans 925 acres in ARC-Co, Wheat 175 Acres in ARC-CO.   Later on when I did the ECO/SCO video, I had to move those 925 Bean Acres out of Arc-Co and back into PLC.   When I tried to do that I get an error message.  "Please enter a valid number, the two nearest valid values are 900 and 1000"   So it won't let me enter 925 Bean Acres in PLC due to this constraint on "valid values".   I will put it at 900 for now in the model and then you can access my Madison County farm and you'll see the problem if you try to replace the 900 PLC acres with 925.

**Comment by ddrake** [comment]
This bug has been lurking since the update that set reasonable up/down numeric control increments.  The validation error was generated by the browser.  We never want the browser to perform validation so all the form classes set a "novalidate" attribute.  The problem here was that way back when I updated templates for edit forms to use the crispy _form tag_ instead of the crispy _filter_, I missed updating this particular template so the form didn't get the novalidate attribute even though it was set in the form class.  I'll check all the form templates again in case I missed any others.

Fixed in commit: dabd27a
